csspositioning.css

```diff
@@ -0,0 +1,78 @@
+.container {
```

Comment:
Este ok cum ai pozitionat elementele. 
Daca doresti sa obtimizezi un pic lucrurile, poti pune fiecare div unul in altul
`"<div class="box-yellow"><div class="box-blue"><div class="box-green"></div></div></div>"`
Astfel, in loc sa te complici cu a calcula cati px sa impingi stanga/sus, poti asa sa pui doar top:50% left:50%
Pentru ca o sa impinga elementul 50% fata de parinte (care este si el o cutie fix cu aceasi dimensiune)


De asemenea, era ok sa creezi o clasa .box{} unde sa pui toate elementele specifice unui box
`
.box{
    width: 300px;
    height: 300px;
    position: absolute;
}
`
Si apoi clasele specifice pentru fiecare cutie. Astfel elemetul div ar fi aratat:
`<div class="box box-yellow">`